File: Frontend/src/types/product.ts
export interface Product {
  id: number;
  name: string;
  description: string;
  category: string;
  price: number;
  status: string;
  createdAt: string;
}

export interface ProductPayload {
  name: string;
  description: string;
  category: string;
  price: number;
  status: string;
}

File: AI.CustomerSupport.API/AI.CustomerSupport/Models/Product.cs
namespace AI.CustomerSupport.API.Models
{
    public class Product
    {
        public int Id { get; set; }

        public string Name { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public string Category { get; set; } = string.Empty;

        public decimal Price { get; set; }

        public string Status { get; set; } = "Active";

        public DateTime CreatedAt { get; set; }
    }
}
